### DISCUSSION TITLE
When 4.2 is released, keep the old eevee and the new eevee

### DISCUSSION THREAD

**Discussion by BUGProtectionAssociation** [Bforartists/Bforartists#4394] [Ideas]
**Is your feature request related to a problem? Please describe.**
When 4.2 is released, keep the old eevee and the new eevee

Don't kill old eevee
**Describe the solution you'd like**
keep the old eevee
**Describe alternatives you've considered**
Stupid developers removed the old eevee, which created a big compatibility gap. A lot of people still need this old eevee and the new features of 4.2

But we need all
**Additional context**
This can be used as a final fallback version when the user can't find the old eevee
The new eevee and the old eevee can be together
Why kill it brutally?

**Comment by ReinerBforartists** [comment]
Hey BugProtectionAssociation,

I have converted the issue to a conversation so that it does not get lost.

We are well aware of the trouble. And we doubted the decision and have thought about to keep it alive too. But we simply miss the man power to keep this legacy part of the code alive and working. I hope you understand it. 

I think that the Blender developers are in the same dilemma here. You need to maintain what you keep. Maybe there should have been a better way of backwards compatibility. Not a hard cut. But sometimes such a hard cut is necessary.

I suggest to bring the protest to the Blender developers. Maybe they know a way to enable a better backwards compatibility :)

Kind regards
Reiner

**Comment by Draise14** [comment]
I ran two polls in Xwitter and Discord. Consensus on Xwitter was pro keeping it, Discord (more active users) was con. This has proven that the decision by vote is inconclusive. 

I talked to the developers from Goo Engine (fork of EEVEE-Legacy) and we both agreed the benefits were less than directly developing what was needed with EEVEE Next. They are also working NPR nodes which may come eventually to Next. 

This coming version has only a couple of missing features:
- Clipped alpha shadows which can be done in shader with greater than/less than math nodes on hashed/dithered }
- Refraction (screen space refraction) to shader to RGB
- Shadow Maps (now defined with soft gradient shadows and are ray traced)
- Contact Shadows (now defined by light settings since shadows are ray traced)
- Shadow bias
- Built in Bloom (can be done with the bloom node in the real time compositor)

And a handful of other descrepencies, but in general quality can be on par or superior. Volumes are faster, smoother, better. Shadows are sharper, more realistic and potentially less buggy and finnicky, HDRI's now cast shadows and lights, you get screen space bounced light and color bleed and much more, better SSS, better PBR model, better refraction, better transmission, better reflections... and the list goes on. Also.. faster compile speeds too. 

So.. bare with the evolution, maybe skip 4.2 and go to 4.3 then you will be on production par. 

Tasks that might be good to follow and "push" for feature parity:
https://projects.blender.org/blender/blender/issues/123020#issuecomment-1209841
https://projects.blender.org/blender/blender/issues/123017#issuecomment-1209702
